sphinxcontrib/openapi/openapi30.py

```diff
@@ -164,6 +165,11 @@ def _example(media_type_objects, method=None, endpoint=None, status=None,
         except (ValueError, KeyError):
             status_text = '-'
 
+    # Provide request samples for GET requests
+    if method == 'GET' and not media_type_objects:
```

Comment:
Why `not media_type_objects` part is needed?

Reply:
You are right, this is probably not needed.